docassemble/PDFStats/pdfstats.py

```diff
@@ -208,275 +181,31 @@ def get_class(k, val=None):
-    </h2>
-    <div id="flush-collapseFour" class="accordion-collapse collapse" aria-labelledby="flush-headingFour" data-bs-parent="#fullTextAccordion">
-      <div class="accordion-body">
-      <p><b>Note: this is a list of each metric and its specific score; they are summed together to get the total score</b></p>
-      <table class="table">
-      <thead>
-        <tr>
-          <th>Metric name</th>
-          <th>Original value</th>
-          <th>Weighted value</th>
-        </tr>
-      </thead>
-      <tbody>
-        { chr(10).join(['<tr><th scope="row">{}</th><td>{:.2f}</td><td>{:.2f}</td></tr>'.format(v[0], v[1], v[2]) for v in lit_explorer._form_complexity_per_metric(stats)]) }
-      </tbody>
-      </table>
-      </div>
-    </div>
-  </div>
-
-</div>
-<br/>
-
-
-<a class="btn btn-primary" href="/pdfstats" role="button">Upload a new PDF</a>
-
-
-<br/>
-
-<p>Rate My PDF is a project of the <a href="https://suffolklitlab.org">Suffolk LIT Lab</a>
-It is part of our broader <a href="https://suffolklitlab.org/docassemble-AssemblyLine-documentation/">Document
-Assembly Line</a> project along with our <a href="https://suffolklitlab.org/form-explorer/">Form
-Explorer</a>.
-</p>
-<p>The results listed here, especially our "time to complete" scores, are in an experimental
-status and should not be relied on. We welcome feedback to improve our scoring!</p>
-
-<p>Feedback? Email <a href="mailto:massaccess@suffolk.edu">massaccess@suffolk.edu</a></p>
-
-<br/>
-
-<p>*: These suggestions are provided by <a href="https://openai.com/blog/gpt-3-apps/">OpenAI's GPT3</a>.</p>
-
-</main>
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.3/jquery.min.js"></script>
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-u1OknCvxWvY5kfmNBILK2hRnQC3Pr17a+RTT6rIHI7NnikvbZlHgTPOOmMi466C8" crossorigin="anonymous"></script>
-    <script>
-    (function($){{
-        $(document).ready(function() {{
-            var accordSec = window.location.hash;
-            if (accordSec.length) {{
-                $(accordSec).collapse("show");
-            }}
-        }});
-     }})(jQuery);
-    </script>
-</body>
-
-</html>
-    """
+    if stats.get("number of passive voice sentences") and stats.get("number of sentences"):
+      passive_percent = int(stats["number of passive voice sentences"]) / stats["number of sentences"] * 100
+    else:
+      passive_percent = 0
+    vars = {
+      "stats": stats,
+      "passive_percent": passive_percent,
+      "title": stats.get('title', file_hash),
+      "complexity_score": formfyxer.form_complexity(stats),
+      "word_count": word_count,
+      "word_count_per_page": word_count/float(stats.get("pages") or 1.0),
+      "difficult_word_count": textstat.difficult_words(stats.get("text")),
+      "get_class": get_class,
+      "get_data":  get_data,
+      "minutes_to_hours": minutes_to_hours,
+      "file_hash": file_hash,
+      "int": int,
```

Comment:
Should probably make some of these functions Jinja filters if possible, but not a priority.